### ISSUE TITLE
Add term rib spur

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10415]
**Preferred term label:**
Rib spur

**Synonyms**
Spurring of rib

**Definition (free text, please give PubMed ID)**
A bony projection (spur, osteophyte) originating from the rib
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/cfcd0f5b-0711-4b16-bb06-77358744e28f) CT chest
PMID: 36388615

**Parent term (use hpo.jax.org/app)**
Abnormal rib morphology HP:0000772

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, PMID: 36388615, PMID: 18949442, 
Spondylocostal dysostosis/Jarcho-Levine syndrome (omim277300) PMID: 26167215

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000652
- label: Rib spur
- parents: [Abnormal rib morphology (HP:0000772)](https://hpo.jax.org/app/browse/term/HP:0000772)
- pmid: PMID:36388615
- def: A bony projection (spur, osteophyte) originating from a rib.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10415
- synonyms:
